lib/rules/disallow-keywords-in-comments.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Disallows keywords in your comments, such as TODO or FIXME
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * #### Example
+ *
+ * `"disallowKeywordsInComments": true`
+ *
+ * Invalid:
+ * ```
+ * // ToDo
+ * //TODO
+ * /** fixme *\/
+ * /**
+ *  * FIXME
+ *  *\/
+ * ```
+ */
+
+var assert = require('assert');
+
+var r = /\b(todo|fixme)\b/i;
+
+module.exports = function() {};
+
+module.exports.prototype = {
+    configure: function(disallowKeywordsInComments) {
+        assert(
+            disallowKeywordsInComments === true,
+            'disallowKeywordsInComments option disallows certain keywords in your comments'
```

Comment:
This error message should say what the required value is. See other rules for suggestions.